source/expressions.tex

```diff
@@ -4157,11 +4157,13 @@
 of some class \tcode{C} with type \tcode{T}, the result has type ``pointer to member
 of class \tcode{C} of type \tcode{T}'' and is a prvalue designating \tcode{C::m}.
 \item
-Otherwise, if the operand is an lvalue of type \tcode{T}, the result has type ``pointer to
-\tcode{T}'' and is a prvalue that is the address of the designated object\iref{intro.memory}
-or a pointer to the designated function. \begin{note} In particular, the address of an
-object of type ``\cv{}~\tcode{T}'' is ``pointer to \cv{}~\tcode{T}'', with the same
-cv-qualification. \end{note}
+Otherwise, for an lvalue operand of type \tcode{T},
```

Comment:
Now that we've applied (as a separate PR) the change to use bullets here with a final "Otherwise", I think we want to go back to "Otherwise, if the operand is an lvalue of type `T`", because we really do mean "if", not "this is necessarily the case".

Reply:
Fixed.